app/Http/Controllers/ItemController.php

```diff
@@ -32,26 +33,47 @@ public function index()
   }
 
   public function detail(int $id)
-{
-  // return "Hello world";
-  // $items = Item::all();
-  // dd($items);
-  $current_item = Item::find($id);
+  {
+    // return "Hello world";
+    // $items = Item::all();
+    // dd($items);
+    $current_item = Item::find($id);
 
-  // dd($current_item);
+    // dd($current_item);
 
-  $images = $current_item->ItemImage()->first();
+    $images = $current_item->ItemImage()->first();
 
```

Comment:
select *from item_images where item_id IN (select id from items where id = 1) limit 1

Reply:
select *from items a INNER JOIN item_images b ON a.id = b.item_id
where a.id = $id